config/201-clusterrole.yaml

```diff
@@ -17,6 +17,7 @@ kind: ClusterRole
 metadata:
   name: ceph-controller
   labels:
+    duck.knative.dev/source: "true"
```

Comment:
Do we need this in the clusterrole? I was under the impression we just needed it in the source yaml

Reply:
This is not needed. We already have the source observer agg rule here: https://github.com/knative-sandbox/eventing-ceph/blob/main/config/201-clusterrole.yaml#L101

This change results in the source ducktype taking the RBAC rights of the controller, which we do not want.

/hold